src/main/java/org/jboss/xavier/integrations/migrationanalytics/business/VMWorkloadInventoryCalculator.java

```diff
@@ -104,4 +103,20 @@ private VMWorkloadInventoryModel createVMWorkloadInventoryModel(Map vmStructMap)
 
         return model;
     }
+
+    private Long getDiskSpaceList(Map vmStructMap) {
+        // If the VM.used_disk_storage is present use it, if not use VM.DISK[*].size_on_disk
+        try {
+            String usedDiskStoragePath = getExpandedPath(USEDDISKSTORAGEPATH, vmStructMap);
+            Number used_disk_storage = (Number) vmStructMap.get(usedDiskStoragePath);
+            if (used_disk_storage != null) {
+                return used_disk_storage.longValue();
+            }
+        } catch (Exception e) {
+            // In versions previous to 1_0_0 it will fail because there is no such property
```

Comment:
@jonathanvila do you think it is worth adding a log here instead of not doing anything in the `catch`?

Reply:
well, we can add the log, this will only happen when we are processing a payload with a version previous to 1.0.0 version, so probably never will have that log. That's why I didnt add that log.

Do you think it's still worth it to add it ?

Reply:
@jonathanvila I see... but if we never use an old version the Catch will never be executed so I think it might be worth adding a warning log to say that `xavier-integration` is using an old version

Reply:
+1